### DISCUSSION TITLE
Homekit motion notification stops working

### DISCUSSION THREAD

**Discussion by budisek** [koush/scrypted#847] [Q&A]
I use ONVIF Tapo C320WS camera with dummy switch (as doorbell) and two way audio plugin as grouped device. Everything worked fine  until May. There is no object (people, car or package) notification on homekit. Just doorbell notification works correctly.

I have tried to update to last version 0.23.0 then succesfully migrate previous data and update all plugins. But no change. Camera works, doorbell works, two-way audio too. 

What could be wrong?

**Comment by webmdiep** [comment]
Did you update the firmware of the camera? 

Could be a similar problem to the c200 camera where ONVIF (motion detection and thus no hksv) stopped working since a late april update.
https://github.com/koush/scrypted/issues/796

Others in the home assistant integration also having the C320 throw errors 
https://github.com/home-assistant/core/issues/91661#issuecomment-1527558224

**Comment by budisek** [comment]
Tapo app says there is no newer update than 1.3.0 for this camera.
